en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -1,30 +1,37 @@
 # Create Your First REST API Proxy
 
-Choreo is a low-code cloud-native platform for building integration solutions. In this tutorial, you will do the following to explore REST API Proxies in Choreo:
+Choreo is a low-code cloud-native platform for building integration solutions. This tutorial explores API proxies supported by Choreo. An API proxy fronts an API, and it is the contact point for applications that need to consume the API.
+
+To understand API Proxies, let's consider an organization with an existing unmanaged API for its Human Resources Management system. This organization wants to revamp the Human Resources Management system, but the existing API alone cannot ensure security for the backend and the application. To address this, you can create an API proxy and apply the necessary security and access-control policies etc.
+
+
+In this tutorial, you will do the following to explore REST API Proxies in Choreo:
 
 1. Expose an existing API by creating a REST API proxy
 2. Deploy the REST API proxy
 3. Test the REST API proxy to verify its functionality
-4. Manage your REST API proxy by adding rate limiting and security policies and leveraging the platform's API management capabilities.
+4. Manage your REST API proxy by adding access control and security policies and leveraging the platform's API management capabilities.
 
 !!!tip "Before you begin..."
-    Familiarize yourself with REST API Proxies in Choreo. [REST API Proxies](../../develop/components/api-proxies.md) 
+    Familiarize yourself with REST API Proxies in Choreo. [REST API Proxies](../../api-proxies.md) 
 
 
 Let's begin...
+
 ## Step 1: Create a REST API Proxy
 
 You can create a REST API proxy in one of the following methods: 
 
 - Design a new REST API proxy using an existing endpoint. 
-- Importing an OpenAPI specification of the existing API.
+- Importing the OpenAPI specification of an existing API.
 
-In this tutorial, you will be importing an OpenAPI specification and creating a REST API proxy:
+In this tutorial, you will be designing a new REST API proxy using an existing endpoint and creating a REST API proxy:
```

Comment:
you will be designing a new REST API proxy using an existing endpoint and creating a REST API proxy: -> you will be designing a new REST API proxy using an existing endpoint: